test/simplecov_test_helper.rb

```diff
@@ -7,4 +7,11 @@
 # required Ruby >= 2.5.0 and Ruby 2.6.0 was marked for EOL
 SIMPLECOV_MIN_RUBY_VERSION = '2.7.0'
 
-require 'simplecov' if RUBY_VERSION >= SIMPLECOV_MIN_RUBY_VERSION
```

Comment:
Currently for some tests with some Rubies, we hit a "file not found" error, which can be puzzling because it's not obvious that there's a LoadError involved (no backtrace is provided) and passing things such as `--trace` won't reach the subshell used for the env tests.

To repro, simply use Ruby 2.7.5 to perform `bundle exec rake test:env[rails52]`. It's perfectly reasonable that Ruby 2.7.5 might be used in conjunction with Rails 5.2. But while Ruby 2.7+ will satisfy the SimpleCov helper check, the Rails 5.2 test env `Gemfile` does not specify `simplecov`.

Now we simply rescue and report load errors but don't let them stop the tests from running.